api/Pipfile

```diff
@@ -5,7 +5,8 @@ name = "pypi"
 
 [packages]
 cffi = "==1.11.5"
-celery = "==3.1.25"
+celery = {extras = ["sqs"],version = "==4.4.0"}
+kombu = {git = "https://github.com/jonathaningram/kombu.git",ref = "sqs-no-list-queues"}
```

Comment:
bring under govau?

Reply:
we already had a `govau` fork and it just felt easier to do it there ^ but yeah should move it over.